cgmanifest.json

```diff
@@ -158,7 +158,67 @@
         },
         "comments": "CMake for building our CI build"
       }
-    }
+    },
+    {
+      "component": {
+        "type": "git",
+        "git": {
+          "commitHash": "f6a5d4e8eb7476b8d7fc12a56dff300c1c986131",
+          "repositoryUrl": "https://github.com/microsoft/vcpkg"
+        },
+        "comments": "vcpkg for package management"
+      }
+    },
+    {
+      "component": {
+        "type": "git",
+        "git": {
+          "commitHash": "54717d6adb67c15f11b50b0389c426eecb5470da",
+          "repositoryUrl": "https://github.com/triton-inference-server/client"
+        },
+        "comments": "triton client to talk to AML triton service"
+      }
+    },
+    {
+      "component": {
+        "type": "git",
+        "git": {
+          "commitHash": "f54b0e47a08782a6131cc3d60f94d038fa6e0a51",
+          "repositoryUrl": "https://github.com/Tencent/rapidjson"
```

Comment:
When we get packages using vcpkg does it need to specify this commit hash so we're consistent?

Reply:
it is the commit hash from rapidjson 1.1.0, which is consistent with the one we referred in cmake.

Reply:
How does vcpkg select that version and does anything prevent it randomly changing in the future?

Reply:
we are with a fixed version of vcpkg, would package versions from there randomly changing?

Reply:
I would have expected that to be the vcpkg software rather than a fixed version for every single package vcpkg supports.

There are versioning tags for selecting specific versions of packages (at least in the manifest mode).

https://learn.microsoft.com/en-us/vcpkg/reference/vcpkg-json#dependency-version-gt
https://learn.microsoft.com/en-us/vcpkg/users/versioning#baselines